src/Chapter1.hs

```diff
@@ -541,8 +543,11 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z
+    | x > y = mid y x z
+    | y > z = mid x z y
+    | otherwise = y 
```

Comment:
Very elegant recursive solution with the sorting 😍

Reply:
I actually saw it on someone else's solution and was mindblow 🤯